package/__tests__/config.js

```diff
@@ -1,18 +1,17 @@
 /* global test expect, describe */
 
-const { chdirCwd, chdirTestApp } = require('../utils/helpers')
+const { chdirCwd, chdirTestApp, resetEnv } = require('../utils/helpers')
 
 chdirTestApp()
 
 const config = require('../config')
 
 describe('Config', () => {
-  beforeEach(() => jest.resetModules())
+  beforeEach(() => jest.resetModules() && resetEnv())
```

Comment:
`resetEnv` like this might give you more problems than you would expect. I would do:

```js
beforeEach(() => {
  jest.resetModules()
  process.env = {}
  // OR
  // delete process.env.RAILS_ENV
})
```